draft-kazuho-httpbis-priority.md

```diff
@@ -334,12 +334,12 @@ amend the precedence to suit it (see {{merging}}).
 The Priority header field is an end-to-end signal of the request priority from
 the client or the response priority from the server.
 
-As is the ordinary case for HTTP caching ({{?RFC7234}}), a response with a
-Priority header field might be cached and re-used for subsequent requests.
-When an origin server generates the Priority response header field based on
-properties of an HTTP request it receives, the server is expected to control the
-cacheability or the applicability of the cached response, by using header fields
-that control the caching behavior (e.g., Cache-Control, Vary).
+The sole intention of prioritization is to optimize how responses are being
```

Comment:
I'm not sure it's the sole intention of prioritization.  Don't we say you can use this to inform request behavior as well?

Reply:
I think we are trying to say that the intention of prioritization is to optimise HOW responses are being provided, not WHAT is provided.  I started to walk this path in the introduction but didn't want to get too much into HTTP-core leet speak. IIUC what we are trying to say, I think we can riff off the following text fragments from the intro:

```

   To provide meaningful representation of a document at the earliest
   moment, it is important for an HTTP server to prioritize the HTTP
   responses, or the chunks of those HTTP responses, that it sends.
```

``` 
   This document defines the Priority HTTP header field that can be used
   by both client and server to specify the precedence of HTTP responses
```